MongoDB/ mongoCrCollectionValidatorChk.md

```diff
@@ -0,0 +1,117 @@
+# mongoCrCollectionValidatorChk
+
+Every `db.createCollection` statement must include a validator definition statement
+
+regex: `(?is)(?=.*createCollection)^(?!.*validator\s)`
+
+# Sample Passing Script
+``` mongosh 
+{
+            "changeSet": {
+                "id": "create_students-collection",
+                "author": "jennl",
+                "labels": "[jira-101],'version 1.0'",
+                "runWith": "mongosh",
+                "changes": [
+                  {
+                    "mongoFile": {
+                      "dbms": "mongodb",
+                      "path": "create-collection-students.js",
+                      "relativeToChangelogFile": true
+                    }
+                  }
+                  ],       
+                "rollback": [
+                    {
+                      "mongo": {
+                        "mongo": "db.students.drop()"
+                      }
+                    }
+                ]
+            }    
+        }
+--create-collection-students.js contents
+-- current bug in core is not finding QCs violation when this is embedded in changelog so
+--it is defined in separate file that contains what is passed to mongosh
+db.createCollection("students", {
+        validator: {
+                $jsonSchema: {
+                      bsonType: "object",
+                      title: "Student Document Validation",
+                      required: [ "name", "year", "major" ],
+                      properties: {
+                        name: {
+                            bsonType: "string",
+                            description: "name must be a string and is required"
+                        },
+                        year: {
+                            bsonType: "int",
+                            minimum: 2017,
+                            maximum: 3017,
+                            description: "year must be an integer between 2017 and 3017 and is required"
+                        },
+                        major: {
+                            enum: [ "Math", "English", "Computer Science", "History" ],
+                            description: "can only be one of the enum values and is required"
+                        }
+                    }
+                }
+        }
+    });
+
+```
+# Sample Failing Scripts
+``` mongosh 
+{
+            "changeSet": {
+                "id": "create_students-collection",
+                "author": "jennl",
+                "labels": "[jira-101],'version 1.0'",
+                "runWith": "mongosh",
+                "changes": [
+                  {
+                    "mongoFile": {
+                      "dbms": "mongodb",
+                      "path": "create-collection-students.js",
```

Comment:
create-collection-students-fail.js to make it more obvious

Reply:
I have completed this and pushed it to repo